packages/eslint-plugin/src/rules/no-css-prop-without-css-function/index.ts

```diff
@@ -220,7 +221,7 @@ export const noCssPropWithoutCssFunctionRule: TSESLint.RuleModule<string> = {
   meta: {
     docs: {
       url: 'https://github.com/atlassian-labs/compiled/tree/master/packages/eslint-plugin/src/rules/no-css-prop-without-css-function',
-      recommended: 'error',
+      recommended: 'strict',
```

Comment:
TS v6 uses 'strict' instead of 'error'.  This goes back to 'error' in later versions of TS-eslint

Reply:
are there docs i can read about this?

https://typescript-eslint.io/developers/custom-rules/#extra-rule-docs-types implies that `recommended` only takes a boolean value

Reply:
@dddlr it changed to a boolean in the latest version (v8).  Here's the shape of that type in v7: https://github.com/typescript-eslint/typescript-eslint/blob/v7.18.0/packages/utils/src/ts-eslint/Rule.ts#L28

The options for `recommended` in that version are `'recommended' | 'strict' | 'stylistic'` (https://github.com/typescript-eslint/typescript-eslint/blob/v7.18.0/packages/utils/src/ts-eslint/Rule.ts#L9)

Reply:
Removed